lib/r10k/forge/module_release.rb

```diff
@@ -13,6 +13,7 @@ class ModuleRelease
 
       def_setting_attr :proxy
       def_setting_attr :baseurl
+      def_setting_attr :cache_root, File.expand_path(ENV['HOME'] ? '~/.r10k/cache': '/root/.r10k/cache')
```

Comment:
Here, I was following the original git cache settings. Do you want to move this to /tmp if no $HOME?